### ISSUE TITLE
[ERROR]: without any message on Ansible 2.5.1

### ISSUE THREAD

**Issue by arbabnazar** [ansible/ansible#39129]
##### ISSUE TYPE

 - Bug Report

##### ANSIBLE VERSION

```
ansible 2.5.1
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/var/lib/jenkins/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.14 (default, Sep 18 2017, 00:00:00) [GCC 4.8.4]
```

##### CONFIGURATION

ANSIBLE_NOCOWS(/etc/ansible/ansible.cfg) = True
ANSIBLE_PIPELINING(/etc/ansible/ansible.cfg) = True
ANSIBLE_SSH_ARGS(/etc/ansible/ansible.cfg) = -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="~/.ansible/tmp/ansible-ssh-%h-%p-%r" -o ServerAliveInterval=30
DEFAULT_EXECUTABLE(/etc/ansible/ansible.cfg) = /bin/bash
DEFAULT_FORKS(/etc/ansible/ansible.cfg) = 5
DEFAULT_MANAGED_STR(/etc/ansible/ansible.cfg) = This file is created and updated by ansible, edit at your peril
DEFAULT_TIMEOUT(/etc/ansible/ansible.cfg) = 120
DEFAULT_TRANSPORT(/etc/ansible/ansible.cfg) = smart
HOST_KEY_CHECKING(/etc/ansible/ansible.cfg) = False
RETRY_FILES_ENABLED(/etc/ansible/ansible.cfg) = False
##### OS / ENVIRONMENT

Ubuntu 14.04.5 LTS
##### SUMMARY

When I run the playbook, it just give me the `[ERROR]:` without any message, although the playbook run successfully but it's really annoying.

##### STEPS TO REPRODUCE

```yaml
---
- hosts: all
  become: yes
  gather_facts: yes
  vars:
    tomcat_webapps_dir: "/var/lib/tomcat7/webapps"

  tasks:
    - name: Stop the tomcat service
      service:
        name: tomcat7
        state: stopped

    - name: Delete the OLD deployed application version
      file:
        path: "{{ tomcat_webapps_dir }}/{{ item }}"
        state: absent
      with_items:
        - ROOT
        - ROOT.war

    - name: Download the latest version of "{{ SERVICE }}" from S3 bucket
      aws_s3:
        aws_access_key: "{{ AWS_S3_ACCESS_KEY }}"
        aws_secret_key: "{{ AWS_S3_SECRET_KEY }}"
        bucket: "{{ S3_ARTIFACTS_BUCKET }}"
        object: "/{{ SERVICE }}/{{ DEPLOY_APP_VERSION }}/ROOT.war"
        mode: get
        dest: "{{ tomcat_webapps_dir }}/ROOT.war"

    - name: Start the tomcat service
      service:
        name: tomcat7
        state: started
```

##### ACTUAL RESULTS

```
ansible-playbook -i ../inventory/ec2.py --limit tag_Name_staging-app app-deploy.yml -e@/var/lib/jenkins/workspace/deploy-staging-app-from-s3/sysadmin/secure/generic.yml -e@/var/lib/jenkins/workspace/deploy-staging-app-from-s3/sysadmin/secure/staging.yml -e@/var/tmp/extra-vars-2507.yml -u ubuntu '--private-key=~/.ssh/aws.pem' -vvvvv
ansible-playbook 2.5.1
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/var/lib/jenkins/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible-playbook
  python version = 2.7.14 (default, Sep 18 2017, 00:00:00) [GCC 4.8.4]
Using /etc/ansible/ansible.cfg as config file
setting up inventory plugins
 [ERROR]:

Parsed /var/lib/jenkins/workspace/deploy-staging-app-from-s3/sysadmin/ansible/inventory/ec2.py inventory source with script plugin
Loading callback plugin default of type stdout, v2.0 from /usr/local/lib/python2.7/dist-packages/ansible/plugins/callback/default.pyc
```

**Comment by arbabnazar** [comment]
@bcoca is this related issue? https://github.com/ansible/ansible/issues/39007
https://github.com/ansible/ansible/pull/39009

**Comment by geerlingguy** [comment]
I’ve been seeing this too, all day today, only on the machine where I upgraded to 2.5.1, but not on my 2.5.0 machine.

**Comment by arbabnazar** [comment]
@geerlingguy yes, you are right. I have downgraded the `ansible` to `2.5.0` from `2.5.1` using `pip install ansible==2.5.0` and error went away. Hope it will get fixed in future release

**Comment by sivel** [comment]
Closing as a duplicate of #39007

**Comment by mattburns** [comment]
Thanks all. In case anyone reads this and wants to downgrade: `pip install ansible==2.5.0`

**Comment by arbabnazar** [comment]
this error is even not fixed on 2.5.2 @sivel

**Comment by bcoca** [comment]
fix will be released in 2.5.3
